### PR TITLE
[AIRFLOW-2961] Refactor tests.BackfillJobTest.test_backfill_examples test

### PR DESCRIPTION
**Pull request by Fokko**
Simplify this test since it takes up 15% of all the time. This is because every example dag, with some exclusions, are backfilled. This will put some pressure on the scheduler and everything. If the test just covers a couple of dags should be sufficient.

Before:
```
[success] 15.03% tests.BackfillJobTest.test_backfill_examples: 254.9323s
```
Now:
```
[success] 5.39% tests.BackfillJobTest.test_backfill_examples: 79.6985s
```

Make sure you have checked _all_ steps below.

### Jira

- [ ] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-XXX
  - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a Jira issue.

### Description

- [ ] Here are some details about my PR, including screenshots of any UI changes:

### Tests

- [ ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [ ] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
  - When adding new operators/hooks/sensors, the autoclass documentation generation needs to be added.

### Code Quality

- [ ] Passes `git diff upstream/master -u -- "*.py" | flake8 --diff`

### CODE REVIEW

**Comment by Fokko** [comment]
PTAL @bolkedebruin @kaxil @mistercrunch

File: tests/jobs.py [hunk 1]
```diff
@@ -194,30 +195,32 @@ def test_backfill_multi_dates(self):
...
-            if 'example_dags' in dag.full_filepath and dag.dag_id not in skip_dags
+            if 'example_dags' in dag.full_filepath and dag.dag_id in include_dags
         ]
 
         for dag in dags:
             dag.clear(
                 start_date=DEFAULT_DATE,
                 end_date=DEFAULT_DATE)
 
+        # Make sure that we have all dags the dags that we want to test available
```
Comment (kaxil) [inline tests/jobs.py]: Nit: `all dags the dags that we want to` to `all the dags that we want to`

**Review by kaxil — CHANGES_REQUESTED** [review]

File: tests/jobs.py (hunk 1 above)
Comment (Fokko) [reply tests/jobs.py]: Thanks, fixed 👍

**Review by kaxil — APPROVED** [review]
LGTM.

**Comment by kaxil** [comment]
@Fokko Hey, hope you had a long weekend, would you be able to upload the 1.10.0 artifacts to PyPi?

**Comment by kaxil** [comment]
@Fokko Let me know if you are busy, I will upload then :)

**Pull request merged**